hooks.yaml

```diff
@@ -1,6 +1,6 @@
 -   id: autopep8-wrapper
     name: autopep8 wrapper
-    description: Runs autopep8 over python source.
+    description: Runs autopep8 over python source. (You'll want args: [-i] when using this hook, see .pre-commit-config.yaml for an example.)
```

Comment:
this is invalid yaml, you'll need to quote this literal (yuno use pre-commit hooks!)